Exception has occurred: PermissionError
[Errno 13] Permission denied
  File "/home/everton/Desktop/teste.py", line 10, in __init__
    self.socket.bind((self.host, self.port))
  File "/home/everton/Desktop/teste.py", line 64, in <module>
    server = HTTPServer("localhost", 80)
PermissionError: [Errno 13] Permission denied
